### PR TITLE
Flatten terms section

### LINKED ISSUE #4197: Flatten legal pages - terms and policies

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Flattened legal terms-and-policies section
- Updated navigation
- Updated redirects
- Added some special redirects requested by legal team

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- look at the files in the /legal directory and compare to this [spreadsheet](https://docs.google.com/spreadsheets/d/12kuuGgwRqN3k7wlRc4QKIuZifjvxbynizO_LEjKjBq4/edit#gid=0)
* test the link [/legal/terms](http://0.0.0.0:8001/legal/terms) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/terms) redirects also test that [/terms](http://0.0.0.0:8001/terms) redirects as well
* test the link [/legal/intellectual-propery-policy](http://0.0.0.0:8001/legal/intellectual-propery-policy) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/intellectual-property-policy) redirects also test that [/ip-policy](http://0.0.0.0:8001/ip-policy) redirects as well
* test the link [/legal/intellectual-property-policy/2013-05-14](http://0.0.0.0:8001/legal/intellectual-property-policy/2013-05-14) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/intellectual-property-policy/2013-05-14) redirects
* test the link [/legal/online-account-terms](http://0.0.0.0:8001/legal/online-account-terms) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/online-account-terms) redirects
* test the link [/legal/terms-of-service](http://0.0.0.0:8001/legal/terms-of-service) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/terms-of-service) redirects
* test the link [/legal/confidentiality-agreement](http://0.0.0.0:8001/legal/confidentiality-agreement) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/confidentiality-agreement) redirects
* test the link [/legal/developer-terms-and-conditions](http://0.0.0.0:8001/legal/developer-terms-and-conditions) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/developer-terms-and-conditions) redirects
* test the link [/legal/launchpad-terms-of-service](http://0.0.0.0:8001/legal/launchpad-terms-of-service) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/launchpad-terms-of-service) redirects
* test the link [/legal/livepatch-terms-of-service](http://0.0.0.0:8001/legal/livepatch-terms-of-service) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/livepatch-terms-of-service) redirects
* test the link [/legal/jaas-beta-terms-of-service](http://0.0.0.0:8001/legal/jaas-beta-terms-of-service) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/jaas-beta-terms-of-service) redirects
* test the link [/legal/systems-information-notice](http://0.0.0.0:8001/legal/systems-information-notice) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/systems-information-notice) redirects
* test the link [/legal/font-licence](http://0.0.0.0:8001/legal/font-licence) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/font-licence) redirects
* test the link [/legal/privacy-policy](http://0.0.0.0:8001/legal/privacy-policy) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/privacy-policy) redirects also test that [/privacy-policy](http://0.0.0.0:8001/privacy-policy) redirects as well

* test the link [/legal/terms](http://0.0.0.0:8001/legal/terms) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/terms) redirects also test that [/terms](http://0.0.0.0:8001/terms) redirects as well
--
* test the link [/legal/intellectual-propery-policy](http://0.0.0.0:8001/legal/intellectual-propery-policy) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/intellectual-property-policy) redirects also test that [/ip-policy](http://0.0.0.0:8001/ip-policy) redirects as well
* test the link [/legal/intellectual-property-policy/2013-05-14](http://0.0.0.0:8001/legal/intellectual-property-policy/2013-05-14) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/intellectual-property-policy/2013-05-14) redirects
* test the link [/legal/online-account-terms](http://0.0.0.0:8001/legal/online-account-terms) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/online-account-terms) redirects
* test the link [/legal/terms-of-service](http://0.0.0.0:8001/legal/terms-of-service) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/terms-of-service) redirects
* test the link [/legal/confidentiality-agreement](http://0.0.0.0:8001/legal/confidentiality-agreement) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/confidentiality-agreement) redirects
* test the link [/legal/developer-terms-and-conditions](http://0.0.0.0:8001/legal/developer-terms-and-conditions) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/developer-terms-and-conditions) redirects
* test the link [/legal/launchpad-terms-of-service](http://0.0.0.0:8001/legal/launchpad-terms-of-service) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/launchpad-terms-of-service) redirects
* test the link [/legal/livepatch-terms-of-service](http://0.0.0.0:8001/legal/livepatch-terms-of-service) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/livepatch-terms-of-service) redirects
* test the link [/legal/jaas-beta-terms-of-service](http://0.0.0.0:8001/legal/jaas-beta-terms-of-service) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/jaas-beta-terms-of-service) redirects
* test the link [/legal/systems-information-notice](http://0.0.0.0:8001/legal/systems-information-notice) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/systems-information-notice) redirects
* test the link [/legal/font-licence](http://0.0.0.0:8001/legal/font-licence) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/font-licence) redirects
* test the link [/legal/font-licence/differences](http://0.0.0.0:8001/legal/font-licence/differences) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/font-licence/ofl-1.1-ufl-1.0.diff) redirects
* test the link [/legal/font-licence/faq](http://0.0.0.0:8001/legal/font-licence/faq) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/font-licence/faq) redirects
* test the link [/legal/privacy-policy](http://0.0.0.0:8001/legal/privacy-policy) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/privacy-policy) redirects also test that [/privacy-policy](http://0.0.0.0:8001/privacy-policy) redirects as well

## Issue / Card

Fixes #4197

### CODE REVIEW

File: redirects.yaml [hunk 1]
```diff
@@ -251,6 +251,7 @@ contact/whitepaper/?: "https://insights.ubuntu.com/white-papers/"
 containers/docker-ubuntu/?: "/containers"
 containers/kubernetes/?: "/kubernetes"
 containers/lxd/?: "https://linuxcontainers.org/"
+data-privacy/?: "/legal/data-privacy"
```
Comment (anthonydillon) [inline redirects.yaml]: I prefer this URL but the URL is /legal/dataprivacy

File: templates/legal/developer-terms-and-conditions/2016-04-06.html [hunk 2]
```diff
@@ -60,7 +60,7 @@ <h3 id="dev-terms-support">6. Support</h3>
...
+      <p>Canonical may make logos or phrases available to you to publicise your App&#39;s availability through the Client Software. Canonical hereby grants you a worldwide, non-exclusive licence during the Term to use such logos or phrases for the purpose of marketing your App in connection with its availability through the the Client Software, subject to Canonical&#39;s Intellectual Property Rights policy, which Canonical may update from time to time at <a href="https://www.ubuntu.com/legal/intellectual-property-policy">www.ubuntu.com/legal/intellectual-property-policy</a>.</p>
```
Comment (anthonydillon) [inline templates/legal/developer-terms-and-conditions/2016-04-06.html]: Should be a relative link.

File: templates/legal/developer-terms-and-conditions/2018-07-23.html [hunk 3]
```diff
@@ -81,7 +81,7 @@ <h3 id="dev-terms-trial-versions">7. Trial versions</h3>
         the trial version, then the commercial terms in this Agreement apply.</p>
       <h3 id="dev-terms-marketing">8. Marketing</h3>
       <p>Canonical may make logos or phrases available to you to publicise your App&apos;s availability through the Client Software. Canonical hereby grants you a worldwide, non-exclusive licence during the Term to use such logos or phrases for the purpose
-        of marketing your App in connection with its availability through the the Client Software, subject to Canonical&apos;s Intellectual Property Rights policy, which Canonical may update from time to time at <a href="https://www.ubuntu.com/legal/terms-and-policies/intellectual-property-policy">www.ubuntu.com/legal/terms-and-policies/intellectual-property-policy</a>.</p>
+        of marketing your App in connection with its availability through the the Client Software, subject to Canonical&apos;s Intellectual Property Rights policy, which Canonical may update from time to time at <a href="https://www.ubuntu.com/legal/intellectual-property-policy">www.ubuntu.com/legal/intellectual-property-policy</a>.</p>
```
Comment (anthonydillon) [inline templates/legal/developer-terms-and-conditions/2018-07-23.html]: Should be relative

File: templates/legal/developer-terms-and-conditions/index.html [hunk 4]
```diff
@@ -78,7 +78,7 @@ <h3 id="dev-terms-trial-versions">7. Trial versions</h3>
         the trial version, then the commercial terms in this Agreement apply.</p>
       <h3 id="dev-terms-marketing">8. Marketing</h3>
       <p>Canonical may make logos or phrases available to you to publicise your App&apos;s availability through the Client Software. Canonical hereby grants you a worldwide, non-exclusive licence during the Term to use such logos or phrases for the purpose
-        of marketing your App in connection with its availability through the the Client Software, subject to Canonical&apos;s Intellectual Property Rights policy, which Canonical may update from time to time at <a href="https://www.ubuntu.com/legal/terms-and-policies/intellectual-property-policy">www.ubuntu.com/legal/terms-and-policies/intellectual-property-policy</a>.</p>
+        of marketing your App in connection with its availability through the the Client Software, subject to Canonical&apos;s Intellectual Property Rights policy, which Canonical may update from time to time at <a href="https://www.ubuntu.com/legal/intellectual-property-policy">www.ubuntu.com/legal/intellectual-property-policy</a>.</p>
```
Comment (anthonydillon) [inline templates/legal/developer-terms-and-conditions/index.html]: Should be relative

File: redirects.yaml (hunk 1 above)
Comment (pmahnke) [reply redirects.yaml]: this clean-up will be in my next pr

Comment (anthonydillon) [reply redirects.yaml]: This URL will 404 if you leave it as it is.

File: templates/download/desktop/thank-you.html [hunk 5]
```diff
@@ -205,7 +205,7 @@ <h3 class="p-card--title">Select topics you&rsquo;re interested in</h3>
               </span>
             </div>
             <p class="u-margin-medium--top">
-              <small>In submitting this form, I confirm that I have read and agree to <a href="/legal/dataprivacy/newsletter">Canonical’s Privacy Notice</a> and <a href="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy">Privacy Policy</a>.</small>
+              <small>In submitting this form, I confirm that I have read and agree to <a href="/legal/dataprivacy/newsletter">Canonical’s Privacy Notice</a> and <a href="https://www.ubuntu.com/legal/privacy-policy">Privacy Policy</a>.</small>
```
Comment (anthonydillon) [inline templates/download/desktop/thank-you.html]: This link 404s

File: templates/legal/developer-terms-and-conditions/2016-04-06.html [hunk 6]
```diff
@@ -35,7 +35,7 @@ <h3 id="dev-terms-your-app">2. Your App</h3>
       </ol>
       <h3 id="dev-terms-your-obligations">3. Your obligations</h3>
       <ol type="a">
-        <li class="p-list__item">Prior to submitting an App, you must first test the App to confirm it is compatible with the current released version of Ubuntu (as available from <a href="https://www.ubuntu.com">www.ubuntu.com</a> at the date of your submission). Your App will not be automatically ported to versions of Ubuntu released after your initial submission. When it is time to publish your App for a new release of Ubuntu, you must test compatibility with that release. Your Apps must comply with the Publishing Policy.
+        <li class="p-list__item">Prior to submitting an App, you must first test the App to confirm it is compatible with the current released version of Ubuntu (as available from <a href="">www.ubuntu.com</a> at the date of your submission). Your App will not be automatically ported to versions of Ubuntu released after your initial submission. When it is time to publish your App for a new release of Ubuntu, you must test compatibility with that release. Your Apps must comply with the Publishing Policy.
```
Comment (anthonydillon) [inline templates/legal/developer-terms-and-conditions/2016-04-06.html]: This is neither relative or absolute

File: templates/download/desktop/thank-you.html [hunk 7]
```diff
@@ -205,7 +205,7 @@ <h3 class="p-card--title">Select topics you&rsquo;re interested in</h3>
               </span>
             </div>
             <p class="u-margin-medium--top">
-              <small>In submitting this form, I confirm that I have read and agree to <a href="/legal/dataprivacy/newsletter">Canonical’s Privacy Notice</a> and <a href="https://www.ubuntu.com/legal/privacy-policy">Privacy Policy</a>.</small>
+              <small>In submitting this form, I confirm that I have read and agree to <a href="/legal/dataprivacy/newsletter">Canonical’s Privacy Notice</a> and <a href="/legal/privacy-policy">Privacy Policy</a>.</small>
```
Comment (anthonydillon) [inline templates/download/desktop/thank-you.html]: Don't you mean: /legal/dataprivacy

**Pull request merged**